episode-6.msrt

```diff
@@ -393,8 +387,7 @@
 78
 00:05:06,020 --> 00:05:10,110
 [ru] [саркастически] Эх, радуйся, Кэррот. Похоже, никто не увидит твое чУдное зелье..
-[en] [Version 1:] [sarcastically] Ah, rejoice Carrot. It seems nobody will see your wondrous potion...
-[en] [Version 2:] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
+[en] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
```

Comment:
The only time I've ever heard someone say "rejoice" is in church. Nobody would use that in casual conversation unless they were mocking the formality of it. "Rejoice, for we have received bagels on this Friday morning" is how that would be used in a mocking way.

Alternatives would be:

"Oh cool, Carrot" = Informal
"Whew, Carrot" = Relief
"Oh nice, Carrot" = Expressing relief and pleasure with the results

Though I think "You're lucky, Carrot" shows the relief, and is more conversational with what's going on in the scene.